.github/workflows/tex-tests.yml

```diff
@@ -27,16 +27,16 @@ jobs:
         # For example -- os: [ubuntu-latest, macos-latest, windows-latest]
         os: [ubuntu-latest, macos-latest, windows-latest]
 
-        # Test python 3.8, 3.9, and 3.10 by default.
-        python-ver: [8, 9, 10]
+        # Test python version 3.x
+        python-ver: [9, 10, 11]
 
         # Specify which tox environments to test in this list.
         # tox-env: [cov, alldeps, devdeps, astropylts]
         tox-env: [alldeps]
     steps:
-    - uses: actions/checkout@v3
+    - uses: actions/checkout@v1
```

Comment:
Why is this being downgraded to v1? Would upgrading to v4 (the current recommendation by github) suffice too?

Reply:
See this pull request where I upgrade these to fix a deprecation warning https://github.com/PAHFIT/pahfit/pull/253

Reply:
Will do.